CJSpohn/whats-cooking-refactor
# Refactor tests and styling

**Is this a fix or a feature?**

Fix and feature

**What is the change/fix?**

-Fix styling and responsiveness in HTML and .scss
-Add and refactor test blocks in User, Recipe, and Cookbook classes 

**Where should the reviewer start?**

-Open project in your text editor to view code

**Issues** 

-cookbook-test.js is not running (added to issues tickets)

**How should this be tested?**

-go to file directory for the project and run npm start to run tests
-open http://localhost:8080/ in browser to view website and click on items to check current functionality
